website/content/en/docs/faq.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "FAQs"
+linkTitle: "FAQs"
+weight: 90
+---
+## General
+
+### How does a provisioner decide to manage a particular node?
+Karpenter only provisions nodes for pods with a status condition `Unschedulable=True`
+Karpenter will only take action on nodes that it provisions.
+All nodes launched by Karpenter will be labeled with `karpenter.sh/provisioner-name`.
+
+### What cloud providers are supported?
+AWS is the first cloud provider supported by Karpenter, although it is designed to be used with other cloud providers as well.
+See [[Cloud provider]({{< ref "/docs/concepts/#cloud-provider" >}}) for details.
+
+### What deployment methods are supported by Karpenter?
+To deploy Karpenter manually, see [[Getting Started]({{< ref "/docs/getting-started/" >}}).
+To deploy using Terraform, see [[Getting Started with Terraform]({{< ref "/docs/getting-started-with-terraform/" >}}).
+To deploy using kOps, see [[Getting Started with kOps]({{< ref "/docs/getting-started-with-kops/" >}}).
+
+### Can I write my own cloud provider for Karpenter?
+!!! NEEDS INFO !!!
+
+### What operating system nodes does Karpenter deploy?
+By default, Karpenter uses Amazon Linux 2 images.
+
+### Can I provide my own custom operating system images?
+Karpenter allows you to create your own AWS AMIs using custom launch templates.
+See [Launch Templates and Custom Images]({{< ref "/docs/aws/launch-templates/" >}}) for details.
+
+### Can Karpenter deal with workloads for mixed architecture cluster (arm vs. amd)?
+Yes. Build and prepare custom arm images as described in [Launch Templates and Custom Images]({{< ref "/docs/aws/launch-templates/" >}}).
+Specify the desired architecture when you deploy workloads.
+
+### What RBAC access is required?
+!!! NEEDS INFO !!!
+
+### Can I run Karpenter outside of a Kubernetes cluster?
+!!! NEEDS INFO !!!
```

Comment:
Yes, so long as the controller has access (network and IAM/RBAC) to the k8s API and your provider API